optimize_versioned_sidebars/version-3.10.0-sidebars.json

```
@@ -2107,447 +2107,447 @@
           "href": "/docs/self-managed/zeebe-deployment/zeebe-installation/"
         },
         {
-          "Configuration": [
+          "Zeebe Gateway": [
             {
               "type": "link",
               "label": "Overview",
-              "href": "/docs/self-managed/zeebe-deployment/configuration/"
+              "href": "self-managed/zeebe-deployment/zeebe-gateway/overview/"
```

```suggestion
              "href": "/docs/self-managed/zeebe-deployment/zeebe-gateway/overview/"
```

I'm pretty sure this is the build issue. This link was being treated as relative from the page you were on, so appending `self-managed/zeebe-de.....` to the current URL on all the pages that were failing the link-check.